### PR TITLE
User list

### PR DESCRIPTION
**Pull request by mfmaite**
## Type of change

* [ ] Fix
* [ ] Story
* [x] Chore

## Description of the change

On the Home screen, we want to display the list of the last 20 users. It should display the following information from them:
* Picture ✅
* Name (First + Last) ✅
* Gender ✅
* Date of Birth

While waiting for the API call, we should display a "Loading" message. ✅

Nice to have:
Each Users has a small card with his information and each of them is lined one next to the other. If the page ends, the Card goes to the row below. ✅

## Screenshot/Execution

image

## Related PRs

Links to the related PRs

### CODE REVIEW

File: src/common/user-card/user-card.module.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.usercard {
+  width: 350px;
+  height: 150px;
+  display: flex;
+  flex-direction: row;
+  padding: 15px;
+  border-radius: 10px;
+  border: solid $primary-color-60;
+  margin: 10px;
```
Comment (agustinasalmanton) [inline src/common/user-card/user-card.module.scss:12]: It's needed the margin and the padding at the same time?

File: src/common/user-card/user-card.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  gender?: string,
+}
+
+export const UserCard = ({
+  name, lastname, imageUrl, gender = 'Unkwnown',
+}: UserCardProps) => (
+  <div className={styles.usercard}>
+    <img src={imageUrl} alt="profile" className={styles.profilePicture} />
+    <div className={styles.profileInfo}>
+      <table>
```
Comment (agustinasalmanton) [inline src/common/user-card/user-card.tsx]: the styles.profileInfo can't be in the <table> ?

File: src/pages/user-list/user-list.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+@use '../../assets/stylesheets/colors' as *;
+@use '../../assets/stylesheets/mixins' as *;
+@use '../../assets/stylesheets/variables' as *;
+
+.imgContainer {
+  min-height: $min-height;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (agustinasalmanton) [inline src/pages/user-list/user-list.module.scss]: It's needed the both?

File: src/pages/user-list/user-list.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    let gender: string;
+    if (title === 'ms' || title === 'miss') {
+      gender = 'Female';
+    } else if (title === 'mr' || title === 'mrs') {
+      gender = 'Male';
+    } else {
+      gender = 'Unknown';
+    }
+    return gender;
+  }
```
Comment (agustinasalmanton) [inline src/pages/user-list/user-list.tsx]: Try to use the arrow function notation

File: src/pages/user-list/user-list.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+  return (
+    <div className={styles.container}>
+      {isLoading
+        ? (
+          <div className={styles.imgContainer}>
+            <img src="src/assets/icons/spinner.gif" alt="Spinner" className={styles.spinnerImg} />
+          </div>
```
Comment (agustinasalmanton) [inline src/pages/user-list/user-list.tsx]: What do you think about doing this in a external component, due to we are going to use it in a lots of components?

File: src/common/user-card/user-card.module.scss (hunk 1 above)
Comment (mfmaite) [reply src/common/user-card/user-card.module.scss:12]: @agustinasalmanton Yes, the padding separates the information from the borders and the margin separates the cards between each other

File: src/pages/user-list/user-list.module.scss (hunk 3 above)
Comment (mfmaite) [reply src/pages/user-list/user-list.module.scss]: @agustinasalmanton Yes, because I need the image to be in the center of the page

**Review by agustinasalmanton — APPROVED** [review]

**Pull request merged**